app/models/grda_warehouse/recurring_hmis_export.rb

```diff
@@ -115,10 +115,10 @@ def store(report)
     # re-compress the zip file with a password and 7zip
     private def encrypt_seven_zip(content)
       tmp = Tempfile.new(['hmis_export', '.zip'], 'tmp', binmode: true)
-      local_source_path = tmp.path
+      # local_source_path = tmp.path
```

Comment:
Was this intentional?  Also `app/models/importers/hmis_auto_migrate/uploaded_zip.rb` seems odd

Reply:
Ah yeah I had rubocop errors from merging in release-66 so I fixed them. Unused variables

Reply:
Ah, ok, thanks.  That's not scary at all.